chromadb/test/property/test_embeddings.py

```diff
@@ -138,18 +138,24 @@ def add_embeddings(self, record_set: strategies.RecordSet) -> MultipleResults[ID
             }
             self.collection.add(**normalized_record_set)  # type: ignore[arg-type]
             self._upsert_embeddings(cast(strategies.RecordSet, filtered_record_set))
-            return multiple(*filtered_record_set["ids"])
+            return multiple(*filtered_record_set["ids"] or [])
```

Comment:
`filtered_record_set` shouuld naturally return the right thing here without need for `[]`